### DISCUSSION TITLE
Circle not fading when clicked in classic mod

### DISCUSSION THREAD

**Discussion by superrobibros** [ppy/osu#23241] [Q&A]
### Type

Game behaviour

### Bug description

playing with a legacy skin with classic mod the circle will not fade but just dissapear

https://www.youtube.com/watch?v=jWiShRsxRCM
also loud keyboard

### Version

latest

### Logs

[legacy-ipc.log](https://github.com/ppy/osu/files/11254865/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11254866/network.log)
[performance.log](https://github.com/ppy/osu/files/11254867/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11254868/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11254869/updater.log)
[database.log](https://github.com/ppy/osu/files/11254870/database.log)
[input.log](https://github.com/ppy/osu/files/11254871/input.log)

**Comment by zoey-on-github** [comment]
I'm sorry, but am I missing something here? Because slowing the video allll the way down to .25x shows that circles act the exact same way after you hit them in both mods.

if you're trying to say this doesn't act like stable does, could you please show what it's "supposed" to look like in stable?

**Comment by frenzibyte** [comment]
also refrain from adding unnecessary words in your discussion thread, this is not discord.

**Comment by peppy** [comment]
Going to require more details.

**Comment by monochrome22** [comment]
The circles do fade, they just fade faster, which can make it seem like they just disappear. 
If you don't want them to fade faster, then untick "Fade out hit circles earlier" in Classic mod customization.
